result/src/test/scala/uscala/result/ResultSpec.scala

```diff
@@ -325,10 +325,10 @@ class ResultSpec extends Specification with ScalaCheck {
 
   "sequence for maps" >> {
     "should transform a Map(K -> Fail) into a Fail" >> prop { xs: Map[Int, Int] => xs.nonEmpty ==>
-      (xs.mapValues(Result.fail).sequence must_=== Fail(xs.values.head))
+      (xs.mapValues(Result.fail[Int, Int]).toMap.sequence must_=== Fail(xs.values.head))
```

Comment:
The added `.toMap` in both of these is for the same reason as described at https://github.com/albertpastrana/uscala/pull/15/files#r196741667.